packages/safe-apps-sdk/package.json

```diff
@@ -15,7 +15,7 @@
   ],
   "scripts": {
     "test": "jest",
-    "format-dist": "sed -i '' 's/\"files\":/\"_files\":/' dist/package.json",
+    "format-dist": "sed -i'' 's/\"files\":/\"_files\":/' dist/package.json",
```

Comment:
does this work for you on mac @mikheevm ?

Reply:
https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux

Reply:
yes it works

Reply:
on linux you have to remove `''`

Reply:
https://github.com/gnosis/safe-apps-sdk/tree/master/packages/safe-apps-sdk#build

Reply:
also worked by just removing the space :D